package.json

```diff
@@ -29,15 +29,19 @@
     "format": "npm run format --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next --workspace=@faustwp/cli --workspace=@faustwp/core",
     "dev": "npm run build && npm run dev:next:getting-started",
     "dev:next:getting-started": "rimraf examples/next/getting-started/.next && npm run dev ---workspace=examples/next/getting-started",
-    "docs": "npm start --prefix internal/website",
-    "docs:build": "npm run build --prefix internal/website",
-    "docs:clear": "npm run clear --prefix internal/website",
-    "docs:install": "cd internal/website && npm i && cd ..",
-    "docs:start": "npm start --prefix internal/website",
+    "docs": "npm start --prefix internal/faustjs.org",
+    "docs:build": "npm run build --prefix internal/faustjs.org",
+    "docs:clear": "npm run clear --prefix internal/faustjs.org",
+    "docs:install": "cd internal/faustjs.org && npm i && cd ..",
+    "docs:start": "npm start --prefix internal/faustjs.org",
+    "docs-legacy": "npm start --prefix internal/legacy.faustjs.org",
+    "docs-legacy:build": "npm run build --prefix internal/legacy.faustjs.org",
+    "docs-legacy:clear": "npm run clear --prefix internal/legacy.faustjs.org",
+    "docs-legacy:install": "cd internal/legacy.faustjs.org && npm i && cd ..",
+    "docs-legacy:start": "npm start --prefix internal/legacy.faustjs.org",
     "test": "npm run build && npm test --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next --workspace=@faustwp/core",
     "test:coverage": "npm run build && npm run test:coverage --workspace=@faustjs/core --workspace=@faustjs/react --workspace=@faustjs/next",
-    "start": "npm run serve:prod --prefix internal/website",
-    "wpe-build": "cd internal/website && npm i && cd .. && npm run docs:build",
+    "wpe-build": "exit 1",
```

Comment:
Fail builds so the new docs site does not deploy on `faustjs.org`.

Reply:
Ah, so it should not build docs at all (including locally) as a safety mechanism until our launch? I'm in the branch now attempting to see the new build for `npm run docs` and the other `docs` scripts in `package.lock` and am getting the following in my terminal: 

![image](https://user-images.githubusercontent.com/50935135/193635329-696e1788-2db7-460f-8118-70f856c5c44e.png)

Reply:
You can still build using the `docs` and `docs-legacy` scripts. However, the `wpe-build` script will return an error to not accidently deploy the site until we properly add the root directory in Atlas.

The error you are recieving seems to be because Docusaurus hasn't bee installed yet. You'll need to run `npm run docs:install` or `npm run docs-legacy:install` first.

Reply:
However, it does look like the build is failing 😬 I'll look into that.